### PR TITLE
Feature/login form

### PR DESCRIPTION
**Pull request by UkhachevychT**

### CODE REVIEW

File: ClientApp/src/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+  <link href="//netdna.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" />
```
Comment (Vladacdc) [inline ClientApp/src/index.html:10]: Add bootstrap as a package, please. And why not to use v5.

File: ClientApp/src/app/account/components/register/register.component.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { first } from 'rxjs/operators';
+
+import { AccountService, AlertService } from 'src/app/_services';
+
+@Component({
+//   selector: 'app-register',
```
Comment (Vladacdc) [inline ClientApp/src/app/account/components/register/register.component.ts:9]: Remove commented line

File: ClientApp/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
-  },
-  {
-    path: '**',
-    pathMatch: 'full',
-    redirectTo: 'books'
-  }
+  // {
+  //   path: '',
+  //   pathMatch: 'full',
+  //   redirectTo: 'books'
```
Comment (Vladacdc) [inline ClientApp/src/app/app-routing.module.ts:15]: Remove commented lines

File: ClientApp/src/app/users/components/layout/layout.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  // selector: 'app-layout',
```
Comment (Vladacdc) [inline ClientApp/src/app/users/components/layout/layout.component.ts:4]: Remove commented line

**Review by Vladacdc — CHANGES_REQUESTED** [review]
Mostly minor changes except for bootstrap.

File: ClientApp/src/index.html (hunk 1 above)
Comment (UkhachevychT) [reply ClientApp/src/index.html]: v5 is in beta, let's stick to 4.5.3

**Review by Vladacdc — APPROVED** [review]

**Pull request merged**